src/main/twirl/leaderboard.scala.html

```diff
@@ -115,7 +115,13 @@
               @for(((p, score)) <- participants) {
                 <tr @{ if (session.username.getOrElse("") == p.username) "id=position class=is-selected" else ""}>
                   <td>@{score.rank}</td>
-                  <td class="names" title="@p.username">@p.username</td>
+                  <td class="names" title="@p.username">
+                    @if(finalLeaderboard) {
+                      <a href="/users/submission/@p.username">@p.username</a>
```

Comment:
Can you add the `download` attribute here so that the target is downloaded on click?